tests/specials.py

```diff
@@ -26,6 +26,7 @@
 from functools import partial
 from types import GeneratorType
 from unittest import TestCase
+from asynctest import TestCase as AsyncTestCase
```

Comment:
FYI, added a dependency on this library here. We could implement something similar, but copying their approach would probably result in some sort of licensing issue, as that project uses Apache-2.0.

Reply:
You can add a tests_require key to setup to indicate a test-time-only dependency

Reply:
That worked great and is way better; thanks!